### ISSUE TITLE
Free Fuel from Fuel Cannister 

### ISSUE THREAD

**Issue by snoweyrawr** [henkelmax/ultimate-car-mod#163]
**Bug description**
Players on a server can obtain the fuel container and shift + right click the fuel pump for 500mb of fuel. However they're not charged for this fuel. On an economy server, this could be an issue. I've read the wiki and not seen anything in relation to it and I've checked the settings to see if this issue can be closed or limited.

**Steps to reproduce the issue**
1. Obtain a fuel container and shift+right click on a fuel pump

**Expected behavior**
Grants fuel to the player but doesn't charge them a fee

**Versions**
- Minecraft version 1.18.2
- Mod loader Forge
- Mod loader version 40.2.10
- Mod version 1.0.3

**Comment by henkelmax** [comment]
Does this still happens in newer versions?

**Comment by Finnson11-star** [comment]
yes it still happen. You can also get 100mb fuel for free at any gas stations. If you drive for a bit you can refuel every time with this 100mb. On our server it is very annoying that people don't pay for fuel.

**Comment by Finnson11-star** [comment]
I've also noticed that when you adjust the price at a gas station, you only see the new price after pressing "Start" again. Sometimes the price changes to the number of for example irons stored in the machine after a payment, but refreshing it fixes it. This probably a visual glitch. In multiplayer mode, the price sometimes doesn't appear for others.
